### DISCUSSION TITLE
Redirect to localized content

### DISCUSSION THREAD

**Discussion by sayyad84** [OrchardCMS/OrchardCore#16603] [Q&A]
### Describe the bug
I have used the following code to transfer the content to another language:
`<a rel="alternate" hreflang="en-US" href={{ "en-US" | switch_culture_url }}><button class="ltr">En</button></a>`
but Google shows the following error:
Page with redirect
for example for:
https://t-one-net.com/RedirectToLocalizedContent?targetCulture=fa-IR&contentItemUrl=%2Fen-US%2Fservicedesk-plus
Isn't there a better way to redirect to the desired page that also solves Google's problem?
### Orchard Core version
Orchard Core 1.8.3

**Comment by hishamco** [comment]
It might be related to an old issue I filed #3579
